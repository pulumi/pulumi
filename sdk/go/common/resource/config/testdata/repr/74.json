{"my:special":{"object":{"fizz":"buzz","foo":"bar"},"thing1":1,"thing2":2}}