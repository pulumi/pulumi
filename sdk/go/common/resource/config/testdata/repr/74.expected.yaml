decrypt:
  my:special: '{"object":{"fizz":"buzz","foo":"bar"},"thing1":1,"thing2":2}'
paths:
  my:special:
    value:
      object:
        fizz: buzz
        foo: bar
      thing1: 1
      thing2: 2
    string: '{"object":{"fizz":"buzz","foo":"bar"},"thing1":1,"thing2":2}'
    redacted: '{"object":{"fizz":"buzz","foo":"bar"},"thing1":1,"thing2":2}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABx/4JuAAMGdGhpbmcxBWludDY0BAIAAgZ0aGluZzIFaW50NjQEAgAEBm9iamVjdBdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+CKAACBGZpenoGc3RyaW5nDAYABGJ1enoDZm9vBnN0cmluZwwFAANiYXI=
    secure: false
    isObject: true
  my:special.object:
    value:
      fizz: buzz
      foo: bar
    string: '{"fizz":"buzz","foo":"bar"}'
    redacted: '{"fizz":"buzz","foo":"bar"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAr/4IoAAIEZml6egZzdHJpbmcMBgAEYnV6egNmb28Gc3RyaW5nDAUAA2Jhcg==
    secure: false
    isObject: true
  my:special.object.fizz:
    value: buzz
    string: buzz
    redacted: buzz
    object: ERAABnN0cmluZwwGAARidXp6
    secure: false
    isObject: false
  my:special.object.foo:
    value: bar
    string: bar
    redacted: bar
    object: EBAABnN0cmluZwwFAANiYXI=
    secure: false
    isObject: false
  my:special.thing1:
    value: "1"
    string: "1"
    redacted: "1"
    object: DhAABnN0cmluZwwDAAEx
    secure: false
    isObject: false
  my:special.thing2:
    value: "2"
    string: "2"
    redacted: "2"
    object: DhAABnN0cmluZwwDAAEy
    secure: false
    isObject: false
